server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -982,6 +989,32 @@ private TemplateVariables buildTemplateVariables(CreateFlowInstanceReq flowInsta
                 variables.setAttribute(Variable.CONNECTION_PROPERTIES, entry.getKey(), entry.getValue());
             }
         }
+        Database database = databaseService.detail(flowInstanceReq.getDatabaseId());
+        if (Objects.nonNull(database)) {
+            String environmentName = database.getEnvironment().getName();
+            environmentName =
+                    I18n.translate(environmentName.substring(2, environmentName.length() - 1), null, Locale.US);
+            variables.setAttribute(Variable.ENVIRONMENT_NAME, environmentName);
+            variables.setAttribute(Variable.DATABASE_NAME, database.getName());
+            GetDatabaseOwnerResp databasesOwner =
+                    databaseService.getDatabasesOwner(flowInstanceReq.getProjectId(), database.getId());
+
+            List<Long> ownerIds = databasesOwner.getMembers().stream().map(member -> {
```

Comment:
It won't happen because the data is retrieved using databaseService.getDatabasesOwner(), which populates the project leader as the owner using this method.
done